src/main/java/seedu/address/model/Model.java

```diff
@@ -191,12 +191,22 @@
      */
     void updateFilteredPersonList(Predicate<Person> predicate);
 
+    /**
+     * Wraps the filtered list of persons in a sortedList with a specific comparator
+     */
     void sortPerson(Comparator<Person> comparator);
+
+    /**
+     * Permanently sorts the address book by a specific comparator
+     */
+    void permaSortContacts(Comparator<Person> chooseComparator);
```

Comment:
Consider changing the description to be something more explicit like: "Permanently sorts and edits the underlying save person-list save file" to clearly say that this changes the save file. 

Also consider changing the argument name to just `comparator` instead of `chooseComparator`